[{"raw":{},"required":["Filename","Size"],"package":"python3-pytest-rerunfailures","source":"pytest-rerunfailures","version":"12.0-1","section":"python","priority":"optional","architecture":"all","depends":["python3-importlib-metadata | python3 (>> 3.8)","python3-packaging","python3-pytest (>= 6.2)","python3:any"],"installedSize":62,"maintainer":"Debian Python Team <team+python@tracker.debian.org>","description":"pytest plugin to re-run tests to eliminate flaky failures","homepage":"https://github.com/pytest-dev/pytest-rerunfailures","filename":"pool/main/p/pytest-rerunfailures/python3-pytest-rerunfailures_12.0-1_all.deb","size":16624,"md5":"dac557db3c0ccddebc3ee238a53236fa","sha256":"9f86023bcb2f59932bd325f2a228680d0384dd44c72e5e5ef39e5e1b2f883ce9","descriptionMd5":"b39ddd4149884d1e62c3622245de3f54"}]